library/ssl_tls13_client.c

```diff
@@ -1022,6 +1022,8 @@ int mbedtls_ssl_tls13_write_identities_of_pre_shared_key_ext(
 
     MBEDTLS_SSL_DEBUG_BUF(3, "pre_shared_key identities", buf, p - buf);
 
+    mbedtls_ssl_tls13_set_hs_sent_ext_mask(ssl, MBEDTLS_TLS_EXT_PRE_SHARED_KEY);
```

Comment:
We do this in `mbedtls_ssl_tls13_write_binders_of_pre_shared_key_ext`:
https://github.com/Mbed-TLS/mbedtls/blob/8b6b41f6cd86c70ea92d3ce45b3479b54364f2c8/library/ssl_tls13_client.c#L1061-L1068
The bit mask should be set if the calculation succeeded. Do I miss something?